astropy/coordinates/transformations.py

```diff
@@ -347,23 +347,35 @@ def get_transform(self, fromsys, tosys):
             self._composite_cache[fttuple] = comptrans
         return self._composite_cache[fttuple]
 
-    def lookup_name(self, name):
+    def lookup_name(self, name, suppress_error=True):
         """
         Tries to locate the coordinate class with the provided alias.
 
         Parameters
         ----------
         name : str
             The alias to look up.
+        suppress_error : bool (..versionadded:: 3.0)
```

Comment:
Ah, the `.. versionadded:: 3.0` is a paragraph directive. So you have to put it after the text on a newline:

```
suppress_error : bool (..versionadded:: 3.0)
    If True (the default) then it returns None if the name isn't 
    found otherwise it will raise an Exception.
   
    .. versionadded:: 3.0
```

I would really recommend you try to build the documentation locally and verify that it, indeed, looks okay :)